scripts/create-movies/Movies.csv

```diff
@@ -1,24 +1,24 @@
-Title,Availability,Decade,Description,Handicap,Image,Keywords,Release Year,Runtime,SortTitle,Tomato Score,Created Date,Modified Date,Created By,Unique ID
+Title,Availability,Decade,Description,Handicap,Image,Keywords,Release Year,Runtime,SortTitle,Tomato Score,Created Date,Modified Date,Created By,Unique ID
```

Comment:
@fdlai what changes are included as part of this file? Github isn't doing a great job of showing me what's changed here.

Reply:
Almost all my changes were in the create-movies.js file. There were only 4 other (one line) changes that were not contained in that file, and I mentioned those in the **describe your changes** section in this PR. Just some alterations to models. If you're seeing changes in any CSV file, it must be because of **npm run lint:fix**